Bottom: 887fe53e868d4d7ecf2d2eced22617b30387107f
Top:    2022245d2c5f648f3034d5e1fe2df7f136fd8594
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-07 01:46:16 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
